src/sonic-py-common/sonic_py_common/multi_asic_device_info.py

```diff
@@ -0,0 +1,292 @@
+import glob
+import importlib
+import os
+from os import name
+import sys
+
+from natsort import natsorted
+
+from sonic_py_common.device_info import CONTAINER_PLATFORM_PATH
+from sonic_py_common.device_info import HOST_DEVICE_PATH
+from sonic_py_common.device_info import get_platform
```

Comment:
Since this is from the same package I think you can simplify these to `from .device_info import ...`

Also move these below the swsssdk imports and separate with an empty line. Per[ PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports):

>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.

Reply:
from .device_info import (
    CONTAINER_PLATFORM_PATH,
    HOST_DEVICE_PATH,
    get_platform
)

Reply:
Changed in the latest commit